tests/foreman/api/test_discoveredhost.py

```diff
@@ -243,11 +232,10 @@ def test_create_discovery_rule_1(self):
         @Assert: Host should reboot and provision
 
         @Status: Manual
-
         """
 
     @stubbed()
-    def test_create_discovery_rule_2(self):
+    def test_positive_create_discovery_rule_with_complex_query(self):
```

Comment:
Maybe `test_positive_create_discovery_rule_with_cpu_count`. Let me know if I need to change that code on my side to yours approach